STARTERKIT/js/script.js

```diff
@@ -10,21 +10,15 @@
 // wrapping it with an "anonymous closure". See:
 // - https://drupal.org/node/1446420
 // - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
-(function (Drupal) {
+(function (Drupal, $, debounce, drupalSettings) {
 
   'use strict';
 
   // To understand behaviors, see https://drupal.org/node/756722#behaviors
-  Drupal.behaviors.my_custom_behavior = {
-    attach: function (context, settings) {
-
-      // Place your code here.
-    }
-  };
 
   // We pass the parameters of this anonymous function are the global variables
   // that this script depend on. For example, if the above script requires
   // jQuery, you should change (Drupal) to (Drupal, jQuery) in the line below
   // and, in this file's first line of JS, change function (Drupal) to
   // (Drupal, $)
-})(Drupal);
+})(Drupal, jQuery, Drupal.debounce, drupalSettings);
```

Comment:
If you are already passing `Drupal`, then you don't need to also pass `Drupal.debounce`